watcher/local/watcher.go

```diff
@@ -112,3 +115,16 @@ func newCh(id channel.ID, parent *ch, params *channel.Params) *ch {
 		parent: parent,
 	}
 }
+
+func (w *Watcher) handleEventsFromChain(eventsFromChainSub channel.AdjudicatorSubscription, eventsToClientPubSub adjudicatorPub,
+	thisCh *ch) {
+	for e := eventsFromChainSub.Next(); e != nil; e = eventsFromChainSub.Next() {
+		switch e.(type) {
+		case *channel.RegisteredEvent:
+			log := log.WithFields(log.Fields{"ID": e.ID(), "Version": e.Version()})
+			log.Debug("Received registered event from chain")
+			eventsToClientPubSub.Publish(e)
```

Comment:
Because, in case of local pub-sub, there is no possibility for an error.
The events are just pushed to the channel.

When stop watching, the `eventsFromChainSub` is closed before closing `eventsToClientPubSub`, which eliminates the possibility of writing to a closed channel.